packages/template-retail-react-app/app/components/display-price/index.jsx

```diff
@@ -7,61 +7,193 @@
 
 import React from 'react'
 import PropTypes from 'prop-types'
-import {Skeleton, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
-import {useIntl} from 'react-intl'
-import {useCurrency} from '@salesforce/retail-react-app/app/hooks'
+import {Box, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {defineMessages, useIntl} from 'react-intl'
 
-const DisplayPrice = ({
-    basePrice,
-    discountPrice,
-    isProductASet = false,
-    currency,
-    discountPriceProps,
-    basePriceProps,
-    skeletonProps
-}) => {
+const msg = defineMessages({
+    // price display
+    currentPrice: {
+        id: 'display_price.label.current_price',
+        defaultMessage: '{currentPrice}'
+    },
```

Comment:
If the translation does not have any English text, we don't need to define a message for it. For example: `currentPrice` and `listPrice`.

Reply:
ah good callout. Made the change as suggested